backend/models.py

```diff
@@ -0,0 +1,54 @@
+from sqlalchemy import Column
+from sqlalchemy.types import Integer
+from sqlalchemy.types import String
+from sqlalchemy.types import Boolean
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+class User(db.Model):
+    """
+    User model - for admin
+    """
+    __tablename__ = 'user'
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    username = Column(String(20), unique=True)
+    hashed_password = Column(String(100))
```

Comment:
We are professional enough to now that password need to be hased. So just `password`. We should also choose hashing algorithm and they usually have constant lenght of hash, so it should be put here instead of `100`